experimental/python-flask/stack.yaml

```diff
@@ -0,0 +1,14 @@
+id: python-flask
```

Comment:
Please can you remove the id field here, we no longer need it as it is generated in the `package.sh` script

Reply:
Ok, will do, thanks

Reply:
done